Source/Extensions/Locale+24Hour.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Calendar+24Hour.swift
```

Comment:
This project uses style without file headers, generated automatically by Xcode. Please, remove this header.